.pylintrc

```diff
@@ -281,7 +281,7 @@ ignore-mixin-members=yes
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=
+ignored-modules=test_mwss_submission_data
```

Comment:
why is this ignored?

Reply:
How we deal with test_mwss_submission_data was going to be made into part of a separate card so for this PR thought it would be best to just ignore it. For the command in the description it seems to pick up more lint compared to what our 'run_lint.sh' script does so might have to update it.

Reply:
I think if this is what we are going to do, we need to update the run_lint script.

Reply:
As @shahi645 we need to put this command into `run_lint.sh` otherwise the value will be lost as soon as this PR is closed

Reply:
agreed 👍

Reply:
The only lint rule that should be ignored in test_mwss_submission_data is for the number of asserts. Can we just use a `# pylint: disable=` in the test file?